pkg/network/placement_configuration.go

```diff
@@ -9,15 +9,32 @@ import (
 func GetDefaultPlacementConfiguration() cnao.PlacementConfiguration {
 	return cnao.PlacementConfiguration{
 		Infra: &cnao.Placement{
-			NodeSelector: map[string]string{
-				"beta.kubernetes.io/arch":               "amd64",
-				"node-role.kubernetes.io/control-plane": "",
-			},
-			Tolerations: []corev1.Toleration{
```

Comment:
I think you should keep this and also add back a toleration for node-role.kubernetes.io/master